web/marketing/pages/base.html

```diff
@@ -9,7 +9,7 @@
         </head>
         <body>
             {{template "body" .}}
-            {{template "bootstrap" .}}
+            {{template "bootstrap" .}} /*TODO necessary?*/
```

Comment:
i think this is the design library bootstrap, not our storj bootstrap

Reply:
cool good to know